README.md

```diff
@@ -56,6 +56,25 @@ The **arches_her/install/requirements.txt** file will also need to be edited in
 
 
 
+## Running Arches in a Docker Development Environment
+
+You can also run Arches in a Docker development environment. To do this, pull the `arches` repo and use the two compose files `docker-compose-dependencies.yml` and `docker-compose.yml`.
+
+- Clone the [`arches`](https://github.com/archesproject/arches.git) repository
+- Navigate to the folder where the compose files exist, then execute:
```

Comment:
Worth being a bit more explicit as this is within Arches for HERs so arches_her/arches_her/docker ?  Unless there's a reason not to do that?

Reply:
Done